adcc/testdata/cache.py

```diff
@@ -234,3 +236,27 @@ def adc_states(self):
 
 # Setup cache object
 cache = TestdataCache()
+
+
+def lists_to_ndarray(dictionary):
+    data = dictionary.copy()
+    for key in data:
+        d = data[key]
+        if isinstance(d, dict):
+            data[key] = lists_to_ndarray(d)
+        elif isinstance(d, list):
+            data[key] = np.array(d)
+    return data
+
+
+class QChemData:
+    @property
+    def data(self):
+        thisdir = os.path.dirname(__file__)
+        yaml_file = os.path.join(thisdir, "qchem_dump.yml")
+        with open(yaml_file, "r") as f:
+            data_raw = yaml.safe_load(f)
+        return lists_to_ndarray(data_raw)
```

Comment:
why do you even need this class if all it does is being constructed and then the data property called? Just make it a function.

Reply:
Done.